core/templates/dev/head/domain/state_card/StateCardObjectFactory.js

```diff
@@ -66,6 +80,15 @@ oppia.factory('StateCardObjectFactory', [function() {
     this._destStateName = destStateName;
   };
 
+  StateCard.prototype.setOppiaResponse = function(index, response) {
+    this._inputResponsePairs[index].oppiaResponse = response;
+  };
+
+  StateCard.prototype.setLastOppiaResponse = function(response) {
+    this._inputResponsePairs[
```

Comment:
Same thing -- this.setOppiaResponse(this._inputResponsePairs.length - 1, response)

Reply:
Done.